code/modules/surgery/blood_filter.dm

```diff
@@ -68,7 +68,7 @@
 	if(target.reagents?.total_volume)
 		for(var/datum/reagent/chem as anything in target.reagents.reagent_list)
 			if(!length(bloodfilter.whitelist) || (chem.type in bloodfilter.whitelist))
-				target.reagents.remove_reagent(chem.type, min(round(chem.volume * 0.22, 0.2), 10))
+				target.reagents.remove_reagent(chem.type, clamp(round(chem.volume * 0.22, 0.2), 0.5, 10))
```

Comment:
```suggestion
				target.reagents.remove_reagent(chem.type, clamp(round(chem.volume * 0.22, 0.2), 0.4, 10))
```

Only because we process reagents generally in units of 0.2/0.4.